src/components/common/contact-form.module.scss

```diff
@@ -0,0 +1,50 @@
+@import '../../scss/colors.scss';
+@import '../../scss/breakpoints.scss';
+
+.contact-form {
+  margin-top: 30px;
+
+  input[type='text'],
+  input[type='email'],
+  select,
+  textarea {
+    width: 50%;
+    padding: 12px;
+    border: 1px solid #ccc;
+    border-radius: 4px;
+    box-sizing: border-box;
+    margin-top: 6px;
+    margin-bottom: 16px;
+    resize: vertical;
+    margin-bottom: 50px;
+  }
+
+  @media (max-width: $viewport-md) {
+    input[type='text'],
+    input[type='email'],
+    select,
+    textarea {
+      width: 100%;
+    }
+  }
+
+  .input-header {
+    margin-bottom: 5px;
+    font-weight: 600;
+    font-size: 24px;
+  }
+
+  button {
```

Comment:
Currently, there's a little border for me (FF on Windows):

![image](https://user-images.githubusercontent.com/18607205/79480989-2344f880-7fcc-11ea-8759-db1c250bcd11.png)

I would go with:
- `border: 1.5px solid $color-blueberry-400;` (or similar) or
- `border: none;`